snuba/query_engine.py

```diff
@@ -0,0 +1,201 @@
+import logging
```

Comment:
I'm not sure how I feel about this being a top-level module — I think it implies that the responsibilities of the contents are broader in scope than they are in reality. `raw_query` to me has always felt coupled to `snuba.api` (and not generic query execution) since it deals with things like HTTP status codes. Would it make sense to break up `snuba.api` into `snuba.api.views` and `snuba.api.query` or something? I think the splitter might also make sense to be moved into `snuba.api.split` eventually as well until it's decoupled from `raw_query`.

Reply:
Done, but I did not put views.py under api because I was not able to access templates anymore since snuba was not the root directory of the application and Armin was (hopefully) asleep. So views is in the root directory to simplify the config.